apps/schools/domains/employee/components/changeEmployeeForm/styles/styles.module.scss

```diff
@@ -0,0 +1,26 @@
+.baseRowContainer {
+  .image {
+    @media (max-width: 1700px) {
```

Comment:
^^
regarding the width of the screens, you also need to do this

Reply:
yes, it need to be fixed now